src/log/getLogger.ts

```diff
@@ -53,15 +54,28 @@ class Logger {
 
     if (asyncLocalStorage != null) {
       try {
-        const traceId = asyncLocalStorage.getStore();
-        if (traceId != null) {
-          jsonContext = Object.assign({}, jsonContext, { traceId });
+        const mapOfStuffInLocalStorage = asyncLocalStorage.getStore();
+        if (mapOfStuffInLocalStorage instanceof Map) {
+          const storeId = mapOfStuffInLocalStorage.get('storeId')
+          if (storeId != null) {
+            jsonContext = Object.assign({}, jsonContext, { storeId });
+          }
         }
       } catch (err) {
       }
     }
     
 
+    const span = trace.getSpan(context.active()); 
+    if (span) {
+      const { traceId, spanId } = span.spanContext();
+      console.log(`traceId = '${traceId}', spanId = '${spanId}'`)
+      jsonContext = Object.assign({}, jsonContext, { traceId, spanId })
+    } else {
+      console.log("No Span")
```

Comment:
Does this log in practice? Observe will scape that too, but should be a minority of logs (just startup basically?)